tests/devices/test_default_mixed.py

```diff
@@ -662,6 +663,26 @@ class TestApply:
     """Unit tests for the main method `apply()`. We check that lists of operations are applied
     correctly, rather than single operations"""
 
+    def test_identity(self, tol):
+        """Tests that applying the identity operator doesn't change the state"""
+        num_wires = 2
+        dev1 = qml.device("default.mixed", wires=num_wires)  # prepare basis state
+
+        dev2 = qml.device("default.mixed", wires=num_wires)  # prepare equal superposition
```

Comment:
These seem to be two separate test cases. It's best to either:

a) separate them into two separate cases;
b) parameterize this single test case.

Reply:
Got it